packages/react-composites/src/composites/common/utils.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { _ICoordinates, _useContainerHeight, _useContainerWidth } from '@internal/react-components';
+import { useMemo, useRef } from 'react';
+import { MODAL_PIP_DEFAULT_PX } from './styles/ModalLocalAndRemotePIP.styles';
+
+/**
+ * Interface for ModalLocalAndRemotePIP drag positions
+ */
+interface MinMaxDragPosition {
+  minDragPosition: _ICoordinates | undefined;
+  maxDragPosition: _ICoordinates | undefined;
+}
+
+/**
+ * @private
+ */
+// Use document.getElementById until Fluent's Stack supports componentRef property: https://github.com/microsoft/fluentui/issues/20410
+export const useMinMaxDragPosition = (
+  rtl: boolean | undefined,
+  modalLayerHostElement: HTMLElement | null
```

Comment:
Should both rtl and modalLayerHostElement be required params?
I am not sure if this hook offers value if used without these params?

Reply:
If yes, I would modify the signature to be 

`useMinMaxDragPosition = (modalLayerHostElement: HTMLElement, rtl = false)`

Reply:
Good point, rtl is not needed and can be optional. 
modaLayerHostElement will be needed though. Will make that change! Good catch, thanks Anjul!